service/operator/service.go

```diff
@@ -126,83 +129,46 @@ func New(config Config) (*Service, error) {
 // Boot starts the service and implements the watch for azuretpr.
 func (s *Service) Boot() {
 	s.bootOnce.Do(func() {
-		err := s.tpr.CreateAndWait()
-		if tpr.IsAlreadyExists(err) {
-			s.Logger.Log("debug", "third party resource already exists")
-		} else if err != nil {
-			s.Logger.Log("error", fmt.Sprintf("%#v", err))
-			return
-		}
-
-		s.Logger.Log("debug", "starting list/watch")
+		o := func() error {
```

Comment:
I added the retry support for booting the operator as I think it's useful in general and not just for KVM.